backend/src/modules/player-profiles/core/application/command/PlayerProfile.ts

```diff
@@ -0,0 +1,21 @@
+export class PlayerProfile {
```

Comment:
to pewnie mało ważne ale, żeby było wiadomo, że to komenda to może CreatePlayerProfile

Reply:
aaa wybacz, widzę niżej jest w domain taka klasa...

Reply:
no może jutro ja siądę drugi raz to pozmieniam trochę nazwy tych klas, ale czy coś lepszego wymyślę to nie wiem :-p

Reply:
> to pewnie mało ważne ale, żeby było wiadomo, że to komenda to może CreatePlayerProfile

Mega ważne :) Bo tutaj coś została pomiesza a komenda i brak w ogóle obiektu domenowego.